src/Ryujinx.Ava/UI/Views/Input/RumbleInputView.axaml.cs

```diff
@@ -6,34 +6,34 @@
 using Ryujinx.Common.Configuration.Hid.Controller;
 using System.Threading.Tasks;
 
-namespace Ryujinx.Ava.UI.Windows
+namespace Ryujinx.Ava.UI.Views.Input
 {
-    public partial class RumbleSettingsWindow : UserControl
+    public partial class RumbleInputView : UserControl
     {
-        private readonly InputConfiguration<GamepadInputId, StickInputId> _viewmodel;
+        private RumbleInputViewModel ViewModel;
```

Comment:
```suggestion
        private RumbleInputViewModel _viewModel;
```
Private field should start with `_`